.circleci/config.yml

```diff
@@ -6,24 +6,39 @@ workflows:
       - lint_tests:
           requires:
             - setup
-      - typescript_tests:
-          requires:
-            - setup
       - frontend_tests:
```

Comment:
Above the list of jobs, can you add a detailed explanation with bullet points of how we are thinking about the stages of runs, so that it's easy for someone to understand just by reading that paragraph? That will help provide developers with a "roadmap" for this file.

(Maybe add a link to an image showing the deps, too.)

Reply:
Done! I don't have an image since the workflow link may not be permanent.

Reply:
I've uploaded the image to the wiki: https://github.com/oppia/oppia/wiki/images/circleCIWorkflow.png . You can use this link.

Reply:
Added the link, thanks!